features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Details/components/APIDetailsTopMenu.jsx

```diff
@@ -43,55 +44,62 @@ const styles = theme => ({
     },
 });
 
-const DetailsTopMenu = ({ classes, api, theme }) => {
+const DetailsTopMenu = ({ classes, theme }) => {
     // todo: need to support rev proxy ~tmkb
-    const storeURL = `${window.location.origin}/store-new/apis/${api.id}/overview`;
     return (
-        <div className={classes.root}>
-            <Link to='/apis' className={classes.backLink}>
-                <KeyboardArrowLeft className={classes.backIcon} />
-                <div className={classes.backText}>
-                    BACK TO <br />
-                    LISTING
-                </div>
-            </Link>
-            <VerticalDivider height={70} />
-            <ThumbnailView api={api} width={70} height={50} />
-            <div style={{ marginLeft: theme.spacing.unit }}>
-                <Typography variant='display1'>
-                    {api.name} : {api.version}
-                </Typography>
-                <Typography variant='caption' gutterBottom align='left'>
-                    Created by: {api.provider}
-                </Typography>
-            </div>
-            <VerticalDivider height={70} />
-            <div className={classes.infoItem}>
-                <Typography variant='subheading' gutterBottom>
-                    {api.lifeCycleStatus}
-                </Typography>
-                <Typography variant='caption' gutterBottom align='left'>
-                    State
-                </Typography>
-            </div>
+        <ApiContext.Consumer>
+            {({ api }) => (
+                <div className={classes.root}>
+                    <Link to='/apis' className={classes.backLink}>
+                        <KeyboardArrowLeft className={classes.backIcon} />
+                        <div className={classes.backText}>
+                            BACK TO <br />
+                            LISTING
```

Comment:
Don't we need to localize these strings?

Reply:
Yes we need to. We will do an independent run and fix the missing localization keys in one go.